scripts/cli/package.json

```diff
@@ -49,14 +49,16 @@
     "schema_test": "./util/schema_test.sh",
     "offchain_worker_test": "node ./dist/util/offchain_worker_test.js",
     "polyx_balances": "node ./util/userBalancePOLYX.js",
-    "test": "./test.sh"
+    "test": "npm run clear_cache && jest --runInBand --silent",
+    "clear_cache": "jest --clearCache"
```

Comment:
Since Jest only had tests 0-7.  We still need to run the other integration tests until they have been converted to the Jest format.

Reply:
makes sense alright I will change this shortly.

Reply:
Everything else looks good.  Once the other tests are running, this PR will be ready.  You might have to skip/disable the old 0-7 tests, since each test can only be run once.